app/views/reasonableExcuseJourneys/health/WasHospitalStayRequiredPage.scala.html

```diff
@@ -0,0 +1,25 @@
+
+@import config.AppConfig
+
+@this(layout: Layout,
+        formHelper: FormWithCSRF,
+        govukButton: GovukButton,
+        radios: components.radios)
+
+@(form: Form[_], radioOptions: Seq[RadioItem], postAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageTitle = messages("healthReason.wasHospitalStayRequired.title"),
+    isPageFullWidth = false,
+    formHasErrors = form.errors.nonEmpty,
+    formErrors = form.errors) {
+    @formHelper(action = postAction) {
+        @radios(
+            form = form,
+            legend = messages("healthReason.wasHospitalStayRequired.heading"),
```

Comment:
The text is slightly bigger on ours versus the prototype - could you add this? 
`legendClasses = Some("govuk-fieldset__legend govuk-fieldset__legend--l"),`

Reply:
Same issue with crime, just needed to change the govuk-label in the radio component from xl to l

Reply:
Oh yes! Good spot!